components/barcode/encoding.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Encoding
+page_title: Barcode Encoding
+description: Types of Encoding supported by the Barcode for Blazor.
+slug: components/barcode/encoding
+tags: telerik,blazor,barcode,encoding
+published: True
+position: 3
+---
+
+# Barcode Encoding
```

Comment:
add a basic example. it is easy to put the possible options from the enum in a DDL (we have a kb and several docs articles that do this) so people can see how things work and look. At the very least, say which parametr to set for this.